server/rooms.ts

```diff
@@ -225,14 +243,14 @@ class BasicRoom {
 	getUserList() {
 		let buffer = '';
 		let counter = 0;
-		for (let i in this.users) {
+		for (const i in this.users) {
 			if (!this.users[i].named) {
 				continue;
 			}
 			counter++;
 			buffer += ',' + this.users[i].getIdentityWithStatus(this.id);
 		}
-		let msg = '|users|' + counter + buffer;
+		const msg = '|users|' + counter + buffer;
```

Comment:
```suggestion
		const msg = `|users|${counter}${buffer}`;
```

If you're going to change this line, you might as well convert it to modern string standards.